### ISSUE TITLE
suggestion: make abacus support other rune matrices from other mods (ex thaumic exploration)

### ISSUE THREAD

**Issue by ghost** [BluSunrize/WitchingGadgets#99]
seems abacus only gives information for vanilla thaum rune matrix

would be nice if you had the option / support for inter mod compatiblity with other thaumcraft mods.

there are not THAT many mods that add alternative rune matrices. in fact i think Thaumic Exploration might be the only one.

**Comment by BluSunrize** [comment]
Meh. Effort.
Also: How the heck should I know how they do calculations?

**Comment by ghost** [comment]
@BluSunrize  is identical to regular matrix. in fact i been placing a regular matrix in it's place just to check stability then placing the modified one when doing infusions. all is required is to make any block with Runed Matrix in it be supported or something like that.

**Comment by Shahelion** [comment]
Isn't it Thaumic Horizons, not Exploration, that adds the Modified Runic Matrix?

**Comment by ghost** [comment]
@Zsashas  you are correct. and to re-iterate calculation is IDENTICAL to regular matrix. all? you need to do is add it to the objects than can be right clicked .

**Comment by BluSunrize** [comment]
1) I had no way of knowing it, having never looked at the source and having been given the wrong mod name by you.
2) I'm not working on WG atm. So don't expect this to happen any time soon.

**Comment by ghost** [comment]
I apologize @BluSunrize . It was indeed my mistake. Glad @Zsashas  noticed it. 
I have been placing a regular matrix in the same spot just to do the calculations .
